cluster/ci/origin/images/clean-rpms/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM build-base
+
+ARG SOURCE_URL=https://github.com/openshift/origin.git
+ARG PULL_REFS=master
+
+ENV UMASK=0002
+
+RUN run.sh release-ci cloneref "$SOURCE_URL" && \
+    run.sh bash -c "echo '/cmd/cluster-capacity/go/src/github.com/kubernetes-incubator/cluster-capacity/_output' >> .gitignore" && \
+    run.sh bash -c "echo '/cmd/service-catalog/go/src/github.com/kubernetes-incubator/service-catalog/_output' >> .gitignore" && \
+    run.sh git add .gitignore && \
+    run.sh git commit -m "Modify .gitignore"
```

Comment:
WTF?

Reply:
the rpm build was failing because the _output dirs were dirty. The .gitignore should likely be fixed in origin, but (in logic that seemed to make sense at the time), I thought those components would be split out of origin soon, so it's just as easy to fix up here in the meantime.

Reply:
Let's upstream it, not clear to me we will break it any time soon